Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -190,7 +190,7 @@ protected void testGetActiveParticipantSource(LogEventRequestType request, Boole
         }
 
         assertEquals("NetworkAccessPointID mismatch", ipOrHost, sourceActiveParticipant.getNetworkAccessPointID());
-        assertEquals("SourceActiveParticipant requestor flag mismatch", isRequesting,
+        assertEquals("SourceActiveParticipant requestor flag mismatch", Boolean.TRUE,
```

Comment:
This should be "true" in any case if the audit blob is for the request auditing on initiating Gateway or response auditing on Responding Gateway.